internal/pkg/manifest/errors.go

```diff
@@ -26,7 +26,7 @@ type ErrInvalidPipelineManifestVersion struct {
 }
 
 func (e *ErrInvalidPipelineManifestVersion) Error() string {
-	return fmt.Sprintf("pipeline.yml contains invalid schema version: %d", e.invalidVersion)
+	return fmt.Sprintf("manifest.yml contains invalid schema version: %d", e.invalidVersion)
```

Comment:
Will we throw the same error message for users with the old file structure where their pipeline manifest is named `pipeline.yml` instead of `manifest.yml`?

Reply:
I'll differentiate!